WINDOWS.md

```diff
@@ -0,0 +1,165 @@
+
+This command should terminate with an output similar to:
+
+```
+terrastories_e2e_1 exited with code 0
+```
+
+Open another Powershell window and run the following command to setup:
+
+```
+docker-compose exec web bin/setup
+```
+
+This command runs a setup script that lives in bin/setup, which does:
+
+- install ruby gems
+- install javascript packages
+- setup database
+See the script file for the details.
+
+Enter `localhost:3000` into your internet browser to view the application/
+
+## Using the application 
+
+Everytime you want to open and use the application, make sure you have docker desktop running and run the following command into
+Powershell:
+
+```
+docker-compose up
+```
+
+You can view the running application at `localhost:3000`
+
+It will take a moment to load when first opening the application 
+
+
+## Having troubles? Check our common errors & gotchas
+
+If you run into any problems getting the application to start, please check out a list of common errors & gotchas that we have put together [here](https://docs.google.com/document/d/1uSbQl56rAh3AA8Xm7IRZ8qepAMVN55ZOkAqQ8Kh423E/edit)!
+
+Additionally, feel free to join us in Slack [here](https://t.co/kUtI3lnpW1) and find us in the channel #terrastories :) You can also post an issue and label it with `question`. We will get back to you ASAP!
+
+## Development
+
+Most developer contributions will be focused on the rails app. Because this project uses
+docker, we already have a uniform ruby/rails development environment in our rails docker
+image. Any time you need to run a rails command you should do so from a running docker
+container to take advantage of this consistent environment. Use the following command to
+open a bash console on the rails container:
+
+```
+docker-compose exec web /bin/bash
+```
+
+Now you can treat this console like any other development environment, running rails or
+bundler commands as needed. **Please refrain from running such commands in your local
+environment. Always use the rails container instead.**
+
+Any changes to source files should be made directly in your local filesystem under the
+`/opt/terrastories` directory using your preferred editing tools.
+
+## Backup and restore the Terrastories database
+
+Terrastories stores Places, Speakers, and Stories in a database (Postgres DB). it is possible to back these data up and restore them by running lines of code in a bash terminal. 
+
+Using Powershell:
+
+Backup the DB in PS with:
+
+```
+docker run --rm -v "terrastories_postgres_data:/pgdata" -v "$(pwd):/host" busybox tar -cvzf /host/db-backup-test.tgz -C /pgdata .
+```
+
+Restore a backup in PS with:
+
+```
+docker run --rm -i -v "terrastories_postgres_data:/pgdata" -v "$(pwd):/source/" busybox tar -xvzf /source/db-backup.tgz -C /pgdata
+```
+
+Note: the above code is assuming your build is called `terrastories`. It may be necessary to run `docker volume ls` to get the right Docker container name ending with `_postgres_data`.
```

Comment:
This section seems to be repeated across all of the individual environment setup, I wonder if we could reorganize these sections into the DEVELOPMENT.md file and only leave the split instructions for first-time setup.

Reply:
I agree with @lauramosher's suggestion on this.